geometry/r3_element_body.hpp

```diff
@@ -312,5 +312,19 @@ quantities::Product<LScalar, RScalar> Dot(
   return left.x * right.x + left.y * right.y + left.z * right.z;
 }
 
+inline R3Element<double> BasisVector(int const i) {
```

Comment:
This could be written:
```
return {i == 0, i == 1, i == 2};
```
which is probably more efficient despite the ugly implicit conversions.